Project/Engine/src/sysFrame/System/Console/Console.cpp

```
@@ -0,0 +1,25 @@
+
+    FILE* fp = nullptr;
+    freopen_s(&fp, "CONOUT$", "w", stdout);
+    freopen_s(&fp, "CONOUT$", "w", stderr);
+    freopen_s(&fp, "CONIN$", "r", stdin);
+
+    SetConsoleTitle(System::ConvertString(title).c_str());
+
+    std::wcout << L"Console Initialized" << std::endl;
+    return true;
+#endif
+}
```

cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)